### DISCUSSION TITLE
逻辑重构

### DISCUSSION THREAD

**Discussion by shenjackyuanjie** [MCSLTeam/MCSL2#87] [功能建议] [answered]
https://github.com/LxHTT/MCSL2/blob/3444df943f0f84cb86a5d51c51bbcecf35ebb60b/MCSL2_Libs/MCSL2_Settings.py#L22

你就不能拿 dict.get去做吗
（你懒我找时间发PR

**Comment by lxhtt** [comment, marked as answer]
https://github.com/MCSLTeam/MCSL2/blob/NewUI/MCSL2Lib/settingsPage.py
如何，我这样就避免了多次操作文件导致的可能文件占用，然后程序抽风
另外就直接不用那样了，全部重写了
